lib/pxlogparser.js

```diff
@@ -0,0 +1,24 @@
+const PxConfig = require('./pxconfig');
+
+class PxLogsParser {
+    constructor(config) {
+        this.config = new PxConfig(config, null);
+
+    }
+    enrichLogs(logs) {
+        return logs.map((log) => this.enrichLogRecord(log));
+    }
+
+    enrichLogRecord(log) {
+        log.message = log.message.substring(0, this.config.px_external_logger_max_message_size || log.message.length);
```

Comment:
The px_external_logger_max_message_size should be added to the config or be removed from here, in general it's important but not sure this will be relevant for this enforcer.

Reply:
Removed